sigs.csv

```diff
@@ -0,0 +1,75 @@
+archive
```

Comment:
/hold
@jberkus did you mean to commit this? :x

Reply:
Nope, removed.